service_files/phosphor-broadcast-active-bmc-role.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Broadcast if the BMC role is Active
```

Comment:
The "if" part confuses me a bit here. If this service runs, then the BMC "is" the active right? So service should really be "Broadcast that this BMC role is Active'